_sass/_variables.scss

```
@@ -8,6 +8,9 @@ $color-nav: #444;
 $color-spacer: #eee;
 $color-submenu-bg: white;
 $color-bg: #fafafa;
+$color-eclipse: #3d3d3d;
+$color-gray59: #969696;
```

As for the color of the rest of the text, while using our new background color, the most-muted (least-contrasting) color (with repeating two-digit hex pair) we can use is `#707070`.

Proofs:
https://www.color-hex.com/color/707070
https://webaim.org/resources/contrastchecker/?fcolor=707070&bcolor=f5f5f5&api

```suggestion
$color-gray44: #707070;
```